qiskit/tools/jupyter/qiskit_toolbar/main.js

```diff
@@ -0,0 +1,37 @@
+define([
+    'base/js/namespace',
+    'base/js/events'
+], function (Jupyter, events) {
+
+    // Adds a cell above current cell (will be top if no cells)
+    var add_cell = function () {
+        Jupyter.notebook.
+            insert_cell_above('code').
+            // Define default cell here
+            set_text(`import numpy as np
+# Qiskit
+from qiskit import *
+from qiskit.tools.visualization import plot_histogram
+from qiskit.converters import circuit_to_dag
+from qiskit.transpiler import transpile
+from qiskit.tools.jupyter import *
+# Visualization
+import matplotlib.pyplot as plt
+%matplotlib inline`);
+        Jupyter.notebook.select_prev();
+        Jupyter.notebook.execute_cell_and_select_below();
+    };
+    // Button to add
+    var QiskitButton = function () {
+        Jupyter.toolbar.add_buttons_group([
+            Jupyter.keyboard_manager.actions.register({
+                'help': 'Qiskit functions',
+                'icon': "fa-rocket",
```

Comment:
@lerongil can you get us a "Qiskit" icon here?

Reply:
This is a bit tricky.  One first needs to have the qiskit logo as a symbol font with accompanying css, like FontAwesome, and then it can be done.iconmoon (https://icomoon.io/) is one such service that would do that.